### ISSUE TITLE
Groupname inconsistent after naming convention change

### ISSUE THREAD

**Issue by maowerner** [maowerner/sLapH-contractions#51]
We changed the names from + -> c. The filenames are all changed but at least C3c still has a + in the groupname

**Comment by kostrzewa** [comment]
Indeed, found the culprit. Fixing. Should I also rename the integration testing files?

**Comment by kostrzewa** [comment]
I've also changed all the example input files accordingly

**Comment by kostrzewa** [comment]
I guess renaming the integration test files will not be sufficient...

**Comment by kostrzewa** [comment]
see #52
